app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
check this method once in devise gem and see what is actually

```
:database_authenticatable, :registerable, :recoverable, :rememberable, :validatable, :confirmable
```

Reply:
lets include lockable
lets check what is trackable

Reply:
done